GetIntoTeachingApiTests/Models/TeachingEventRegistrationRequestTests.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace GetIntoTeachingApiTests.Models
+{
+    public class TeachingEventRegistrationRequestTests
+    {

```

Comment:
is this needed?

Reply:
It's not needed at the moment, but I've been creating the corresponding test files for all classes just so they are there, even if there is nothing to test yet (I'm not testing getters/setters).